### PR TITLE
Replace jquery with standard JS

### PR DESCRIPTION
**Pull request by korol-sas**

### CODE REVIEW

**Comment by korol-sas** [comment]
https://github.com/jhund/filterrific_demo/pull/25

File: app/assets/javascripts/filterrific/filterrific.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+      if(removed()){ // if removed clear the interval and don't fire the callback
+        if(ti) clearInterval(ti);
+        return;
+      }
+      var val = input.value;
+      if(prev != val){
+        prev = val;
+
+        // invokes the callback on $this
+    		if (callback && typeof callback === 'function') {
```
Comment (jhund) [inline app/assets/javascripts/filterrific/filterrific.js:131]: Can you replace the /tab characters with two spaces?

File: app/assets/javascripts/filterrific/filterrific.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+	return elements;
+};
+
+Filterrific.observe_field = function(inputs_selector, frequency, callback) {
+  console.log('observe_field 2')
+  frequency = frequency * 1000;
+
+  document.querySelectorAll(inputs_selector).forEach(input => {
+    console.log('observe_field 2 #this =>', input);
```
Comment (jhund) [inline app/assets/javascripts/filterrific/filterrific.js:115]: Can you please remove all console.log statements?

File: lib/filterrific/version.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
 
 module Filterrific
-  VERSION = "5.2.2"
+  VERSION = "5.2.2.2"
```
Comment (jhund) [inline lib/filterrific/version.rb:4]: Let's leave the version as is. I will update it when I release a new version. Unless you need to change the version?

**Pull request merged**